CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ All notable changes to this project will be documented in this file.
 
 ### Features
 
+- *(client)* Identify several other GitHub token types as secrets 
```

Comment:
This will be overwritten when we next generate the changelog (with git cliff), but I appreciate the thought 😊